packages/wrangler/scripts/bundle.ts

```diff
@@ -1,14 +1,17 @@
 import fs from "node:fs/promises";
 import path from "node:path";
-import { build, context } from "esbuild";
+import * as esbuild from "esbuild";
 import { EXTERNAL_DEPENDENCIES } from "./deps";
-import type { BuildOptions } from "esbuild";
+import type { BuildOptions, Plugin, BuildContext } from "esbuild";
 
 // the expectation is that this is being run from the project root
 type BuildFlags = {
 	watch?: boolean;
 };
 
+const WATCH = process.argv.includes("--watch");
+const TEMPLATES_DIR = path.join(__dirname, "../templates");
```

Comment:
These were only used in one place...

Reply:
For a moment they were used multiple times 😆